File: UNIT 2/C/ases4.c
#include <stdio.h>

int main() {
    int hari;
    printf("Masukkan jumlah hari keterlambatan: ");
    scanf("%d", &hari);

    if (hari == 0) {
        printf("Tidak ada denda.\n");
    } else if (hari < 7) {
        printf("Denda 5000 rupiah.\n");
    } else if (hari >= 7 && hari <= 14) {
        printf("Denda 10000 rupiah.\n");
    } else {
        printf("Denda 20000 rupiah.\n");
    }

    return 0;
}

File: UNIT 2/py/ases4.py
hari = int(input("Masukkan jumlah hari keterlambatan: "))

if hari == 0:
    print("Tidak ada denda.")
elif hari < 7:
    print("Denda 5000 rupiah.")
elif 7 <= hari <= 14:
    print("Denda 10000 rupiah.")
else:
    print("Denda 20000 rupiah.")
